Makefile

```diff
@@ -4,7 +4,7 @@ export GOPROXY=https://proxy.golang.org
 
 # Which github repository and branch to use for testing with skopeo
 SKOPEO_REPO = containers/skopeo
-SKOPEO_BRANCH ?= master
+SKOPEO_BRANCH ?= main
```

Comment:
Uh, that was apparently left over — I guess we were saved by a redirect.

But in this case the 5.19 version of c/image needs to be testing with a 5.19-using version of Skopeo, something like `v1.6.2.

Reply:
hmm, so IIUC, that would be handled in `SKOPEO_CIDEV_CONTAINER_FQIN: "quay.io/libpod/skopeo_cidev:${IMAGE_SUFFIX}"` ?? I see that the IMAGE_SUFFIX is a quay.io repo tag and I don't think there's an easy mapping of repo tag <-> skopeo version, or am I looking in the wrong place?

/cc @cevich

Reply:
Oops, my mistake, `SKOPEO_REPO` and `SKOPEO_BRANCH` seems to be completely unused now.

The right variable is `SKOPEO_CI_TAG` in `.cirrus.yml`; that’s consumed in `contrib/cirrus/runner.sh` to check out the relevant Skopeo version.

Reply:
Those unused variables will be fixed on the main branch by https://github.com/containers/image/pull/1541.

Reply:
Ya, I think you're on the right track.  There is no hard correlation between branch, tag, and $IMAGE_SUFFIX.  So long as the `$IMAGE_SUFFIX` is consistent between VMs and container images, as was tested when the `$IMAGE_SUFFIX`.  The tricky thing for release branches is, we cannot easily go back and build new "old" images (for example if they were to get pruned due to disuse).  Hopefully that makes sense.